### DISCUSSION TITLE
diary/2023/12/02/time-management-with-rize/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#151] [Comments]
# diary/2023/12/02/time-management-with-rize/

키워드 : 시간 관리 엑셀, Rize 앱, 시간 관리 잘하는 법, 시간 관리 방법, 시간 관리 잘하는 법, 시간 관리법, 시간 관리 프로그램 시간 관리를 어떻게 하고 계신가요? 시간 관리는 어떤 사람이여도 알고 있으면 좋은 내용 중 하나입니다 학생, 사회초년생, 팀장 등 각자의 상황에 맞는 시간 관리 방법이 필요하고, 저도...

https://zzsza.github.io/diary/2023/12/02/time-management-with-rize/

**Comment by MyType9591** [comment]
경험공유 감사드립니다~ 알려주신 덕택에 RIZE 앱 설치 후 알려주신 코드 입력해서 사용해보고 있습니다. 감사합니다.

**Reply by zzsza** [reply]
댓글 감사합니다! 도움이 되었으면 좋겠네요 :)

**Comment by haeminyim** [comment]
덕분에 rize 잘 쓰고 있습니다. 생산성도 생산성이지만 제가 책상에 앉아있을때의 패턴을 알 수 있어서 너무 좋은 것 같아요. 감사합니다 :)
